x/dualstaking/keeper/hooks.go

```diff
@@ -104,48 +102,7 @@ func (h Hooks) AfterDelegationModified(ctx sdk.Context, delAddr sdk.AccAddress,
 	return nil
 }
 
-// BeforeValidatorSlashed hook unbonds funds from providers so the providers-validators delegations balance will preserve
```

Comment:
put this comment above the new function in `slashing.go`

Reply:
https://github.com/lavanet/lava/pull/1356/commits/68e69157ab644e8c4f9f52fc4e04003fb9251adf